### ISSUE TITLE
Suggestion: Filenames with timestamps and human-readable titles

### ISSUE THREAD

**Issue by leonhandreke** [jethrokuan/org-roam#94]
[Zetteldeft](https://github.com/EFLS/zetteldeft) does this, it just concatenates a timestamp and a title. Maybe this would be the best of both worlds?

**Comment by jethrokuan** [comment]
I'll change the variable `org-roam-file-format` to a function that takes in a single parameter, `title`, that's passed by default. That way users are design the file title format beyond what's available from time.

**Comment by leonhandreke** [comment]
That sounds like a great idea!

(Thanks for making org-roam by the way, I just switched from Zetteldeft and I'm loving it so far!)

**Comment by jethrokuan** [comment]
Any reason for the switch? I didn't use zetteldeft because they weren't real file links, and it broke my publishing flow. But I need to fill up https://org-roam.readthedocs.io/en/develop/comparison/ at some point in time.

**Comment by leonhandreke** [comment]
I only just started using org-roam, but here are some things that I already noticed:

* Real file links, like you said. I want the links to be as future-proof as possible.
* Having the backlinks window always open encourages constantly thinking about the connections. In Zetteldeft, finding backlinks is a more explicit process that opens a new Deft buffer that doesn't live update when switching buffers.
* Auto-create files when inserting a link that doesn't yet exist

No fundamental differences, but these differences make zettels feel a lot more connected in org-roam. I'll let you know once I notice more differences :)

Zetteldeft seems to support Markdown as well (since the links aren't actually real links), which org-roam does not.
